nemo/collections/asr/parts/submodules/rnnt_beam_decoding.py

```diff
@@ -460,7 +466,12 @@ def greedy_search(
 
             # TODO: Figure out how to remove this hard coding afterwords
             while not_blank and (symbols_added < 5):
-                ytu = torch.log_softmax(self.joint.joint(hi, y) / self.softmax_temperature, dim=-1)  # [1, 1, 1, V + 1]
+                if isinstance(self.joint, HATJoint):
+                    ytu, _ = self.joint.joint(hi, y)
```

Comment:
This kinda logic is problematic in the long run. Why not take a bool in the HAT module that determine what self.joint returns - by default it's set and returns both items, otherwise return things in the form of RNNT so that this code doesn't need to change

Reply:
I thought that jit compiler does not like variable outputs numbers. Now I made default mode -- return only logprobs (like the standard rnnt joint) and return both logprobs and internal_lm_logprobs (in case of special boolean flag). This is allowed to save more rnnt decoding code unchanged.